examples/twitter-beam-java/src/main/java/ReadFromTweeterDoFn.java

```diff
@@ -0,0 +1,76 @@
+import org.apache.beam.sdk.io.range.OffsetRange;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.splittabledofn.RestrictionTracker;
+import org.apache.beam.sdk.transforms.splittabledofn.SplitResult;
+import org.joda.time.Duration;
+import org.joda.time.Instant;
+import twitter4j.Status;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.concurrent.BlockingQueue;
+
+@DoFn.UnboundedPerElement
+final class ReadFromTweeterDoFn extends DoFn<String, String> {
```

Comment:
This SDF is missing a watermark estimator. Could you add that too, please?

Here's some info about how to add watermark estimation: https://beam.apache.org/documentation/programming-guide/#sdf-basics

Reply:
Tried to do this, please review.